packages/components/src/components/version-dropdown.tsx

```diff
@@ -0,0 +1,48 @@
+'use client';
+
+import { cn } from '../cn';
+import { Dropdown, DropdownContent, DropdownItem, DropdownTrigger } from './dropdown';
+import { CaretSlimIcon, CheckIcon } from './icons';
+
+export interface VersionDropdownProps {
+  chevronPosition?: 'left' | 'right';
```

Comment:
This isn't supper elegant, but I think it actually make sense.

If we have v1, we'd have this on the right.
If we have "Mesh v1" or "1.0", it makes sense to put it on the left.